Dependencies/protobuf-c/generate-pb-c.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-cd "$(dirname "$0")/../../Source/" && protoc-c --c_out=. "PLCrashReport.proto"
\ No newline at end of file
+cd "$(dirname "$0")/../../Source/" && protoc-c --c_out=. "PLCrashReport.proto" && cd "../Tests/" && protoc-c --c_out=. "PLCrashLogWriterEncodingTests.proto"
```

Comment:
Maybe to separate these two commands? It will be better for readability. Also it's good to have a comment above each line. Do we really need to create two .proto files each time? I mean if we build only PLCrashReporter itself without tests.

Reply:
This script serves an additional purpose: updating the compiled proto-c files whenever our protobuf-c dependency is updated. Its multifunctionality allows it to handle both tasks efficiently. Given that the script's execution time is extremely minimal, I believe there is little value in splitting it into multiple scripts.

Reply:
And we do not run this script automatically during the build, it is executed manually by the contributor only when necessary.